### PR TITLE
`Development`: Export admin users in the monitoring

### PR DESCRIPTION
**Pull request by Hialus**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

#### Server
- [x] **Important**: I implemented the changes with a very good performance and prevented too many (unnecessary) database calls.
- [x] I **strictly** followed the [server coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/server/).
- [x] I documented the Java code using JavaDoc style.

### Motivation and Context

We want to add a way to monitor the current admin users in Artemis, so that it is not missed if someone fraudulently adds one.

### Description

For this I added a MultiGauge to the metrics endpoint exposing this data to e.g. Prometheus. This endpoint is fine for this purpose, as it already is only locked to certain IPs, so no data leakage will occur.
To get all admin users I added a new query in the user repository to only get the logins of the admins,

### Steps for Testing

Prerequisites:
- Local environment

1. Start the server
2. Open the URL "http://localhost:8080/management/prometheus" (Adjusted if you changed the port or smth)
3. Search for `artemis_scheduled_users_admins_active`
4. You should at least find the entry `artemis_scheduled_users_admins_active{admin="artemis_admin",} 1.0`. More if you have more local admin users in Artemis

#### Performance Review
- [x] I (as a reviewer) confirm that the server changes (in particular related to database calls) are implemented with a very good performance
#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

## Summary by CodeRabbit

- **New Features**
  - Enhanced monitoring with new metrics for active admin accounts and release exam student multipliers.

- **Bug Fixes**
  - Improved accuracy in calculating and updating active user metrics.

- **Refactor**
  - Renamed method to better reflect its purpose in calculating active user metrics.

- **Tests**
  - Updated test methods to align with changes in metric calculations.

### CODE REVIEW

File: src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java [hunk 1]
```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
                 .description("Number of exams starting within the next minutes multiplied with students in the course").register(meterRegistry);
     }
```
Comment (SimonEntholzer) [inline src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java:379]: Here you could add some javadoc
```suggestion
    /**
     * Register metrics for active admin accounts
     **/
```

**Review by SimonEntholzer — DISMISSED** [review]
Good addition to be able to monitor active admins :+1:

File: src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java [hunk 2]
```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
...
 
+    private void registerActiveAdminMetrics() {
```
Comment (SimonEntholzer) [inline src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java:380]:
```suggestion
    /**
     * Register metrics for active admin accounts
     **/
    private void registerActiveAdminMetrics() {
```

**Review by marlon-luca-bu — APPROVED** [review]
Code LGTM

File: src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java [hunk 3]
```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
...
+        activeAdminsGauge = MultiGauge.builder("artemis.users.admins.active").description("Number of active admin accounts").register(meterRegistry);
```
Comment (bensofficial) [inline src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java:381]:
```suggestion
        activeAdminsGauge = MultiGauge.builder("artemis.users.admins.active").description("User logins of active admin accounts").register(meterRegistry);
```

**Review by bensofficial — CHANGES_REQUESTED** [review]
The exported admin logins also contain deleted admin users.

**Review by marlon-luca-bu — APPROVED** [review]
Reapprove Code

**Review by magkue — APPROVED** [review]
LGTM, tested in a local environment, works fine.
Remark for other testers: the metric is called `artemis_users_admins_active` not `artemis_scheduled_users_admins_active`

File: src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java [hunk 4]
```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
...
         var startDate = System.currentTimeMillis();
 
         // The authorization object has to be set because this method is not called by a user but by the scheduler
         SecurityUtils.setAuthorizationObject();
 
         cachedActiveUserNames = statisticsRepository.getActiveUserNames(ZonedDateTime.now().minusDays(14), ZonedDateTime.now());
 
+        updateActiveAdminsMetrics();
+
         log.debug("calculateCachedActiveUserLogins took {}ms", System.currentTimeMillis() - startDate);
```
Comment (magkue) [inline src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java:401]: Nit: Also update the log message here to 
```suggestion
        log.debug("calculateActiveUserMetrics took {}ms", System.currentTimeMillis() - startDate);
```

File: src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java [hunk 5]
```diff
@@ -374,21 +377,27 @@ private void registerStudentExamMetrics() {
...
 
+    private void registerActiveAdminMetrics() {
+        activeAdminsGauge = MultiGauge.builder("artemis.users.admins.active").description("User logins of active admin accounts").register(meterRegistry);
+    }
+
     /**
      * Calculate active users (active within the last 14 days) and store them in a List.
      * The calculation is performed every 60 minutes.
      * The initial calculation is done in the constructor to ensure it is done BEFORE {@link #recalculateMetrics()}
      * is called.
```
Comment (magkue) [inline src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java:388]: Nit: Also adapt admins

**Pull request merged**